src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -128,14 +129,14 @@ public static BufferedImage buildCoPowerBar(Commander co, int[] abilityPoints, d
     for( int i = 0; i < abilityPoints.length; ++i )
     {
       int requiredPower = abilityPoints[i];
-      double diff = requiredPower - currentPower;
+      double diff = (requiredPower - currentPower)/Commander.CHARGERATIO_FUNDS;
       atLeastOne |= (diff < 0);
       BufferedImage segment = (diff > 1) ? powerBarPieces.getFrame(0)  // empty
           : ((diff > 0.5) ? powerBarPieces.getFrame(1)                 // 1/3 full
               : ((diff > 0) ? powerBarPieces.getFrame(2)               // 2/3 full
                   : ((slowAnimIndex == 0) ? powerBarPieces.getFrame(3) // filled
                       : powerBarPieces.getFrame(4))) );                // blinking
-      int drawLoc = (requiredPower * 2) - imageBufferW - 3; // -3 to center the image around the power level.
+      int drawLoc = ((int) (requiredPower*powerDrawScaleW)) - imageBufferW - 3; // -3 to center the image around the power level.
```

Comment:
That does sound like a better name, yes.
Seems reasonable to change it along with whatever we eventually decide on the whole discretes/storing charge as funds or stars/scaling stuff, rather than separately.